tests/unit/test_indicator.py

```diff
@@ -0,0 +1,415 @@
+        self._pandas_data = pd.read_csv(str(path.parent)+'/testdata.csv.gz')
+        self._pandas_data['Volume'] /= 1000.0
+        self._cudf_data = cudf.from_pandas(self._pandas_data)
+        warnings.simplefilter('ignore', category=ImportWarning)
+        warnings.simplefilter('ignore', category=DeprecationWarning)
+
+
+    def tearDown(self):
+        pass
+
+    @ordered
+    def test_rate_of_return(self):
+        '''Test rate of return calculation'''
+        r_cudf = rate_of_change(self._cudf_data['Close'], 2)
+        r_pandas = p.rate_of_change(self._pandas_data, 2)
+        err = error_function(r_cudf, r_pandas.ROC_2)
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_trix(self):
+        """ test the trix calculation"""
+        r_cudf = trix(self._cudf_data['Close'], 3)
+        r_pandas = p.trix(self._pandas_data, 3)
+        err = error_function(r_cudf, r_pandas.Trix_3)
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_bollinger_bands(self):
+        """ test the bollinger_bands """
+        r_cudf = bollinger_bands(self._cudf_data['Close'], 20)
+        r_pandas = p.bollinger_bands(self._pandas_data, 20)
+        err = error_function(r_cudf.b1, r_pandas['BollingerB_20'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+        err = error_function(r_cudf.b2, r_pandas['Bollinger%b_20'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_macd(self):
+        """ test the macd """
+        n_fast = 10
+        n_slow = 20
+        r_cudf = macd(self._cudf_data['Close'], n_fast, n_slow)
+        r_pandas = p.macd(self._pandas_data, n_fast, n_slow)
+        err = error_function(r_cudf.MACD, r_pandas['MACD_10_20'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+        err = error_function(r_cudf.MACDdiff, r_pandas['MACDdiff_10_20'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+        err = error_function(r_cudf.MACDsign, r_pandas['MACDsign_10_20'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_average_true_range(self):
+        """ test the average true range """
+        r_cudf = average_true_range(self._cudf_data['High'],
+                                    self._cudf_data['Low'],
+                                    self._cudf_data['Close'], 10)
+        r_pandas = p.average_true_range(self._pandas_data, 10)
+        err = error_function(r_cudf, r_pandas['ATR_10'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_average_true_range(self):
```

Comment:
Duplicated signature for the test. You also have a test named `test_average_true_range` on line 139 above. Rename one of the tests.